courses/free-courses-en.md

```diff
@@ -772,6 +772,8 @@
 * [Learn modern JavaScript](https://scrimba.com/learn/es6) - Beau Carnes (Scrimba) (Scrimba account *required*)
 * [Learn to Program in Javascript: Beginner to Pro](https://www.udemy.com/course/programming-in-javascript) - Raghavendra Dixit (Udemy)
 * [learn:query](https://learnquery.infinum.com)
+* Module Bundlers
```

Comment:
not sure what you meant to do here

Reply:
Based on the previous discussions, I added this in Module Bundlers of Javascript section. Please let me know if I need to update anything